contents/projectManagement/AgileDevelopment.md

```diff
@@ -0,0 +1,127 @@
+# Agile Development
+
+Author: [Joanne Ong](https://github.com/joanneong)
+
+## Overview
+
+In 2015, Forbes published an article boldly titled [Agile: The World's Most Popular Innovation Engine](https://www.forbes.com/sites/stevedenning/2015/07/23/the-worlds-most-popular-innovation-engine/#22b176267c76). Indeed, as a software engineering student, you may have heard of the term "agile development", "agile model" or even simply "agile" before. In this chapter, we will explore agile development in greater depth, and learn why agile development is still thriving today.
+
+## What is agile development?
+
+Popularised in 2001 by the [*Manifesto for Agile Software Development*](http://agilemanifesto.org/), agile development is essentially a software development method that is flexible, iterative and people-centric.
```

Comment:
This sentence structure `Agile development is essentially a software development method that ...` suggests that A is a concrete method like scrum. However, I think agile development is more of an approach to software development as explained by [wikipedia](https://en.wikipedia.org/wiki/Agile_software_development), and from this approach people derive actual concrete development framework. So I think you should change this sentence to reflect that.